apps/mobile/src/app/(home)/settings/wallet/configure/[wallet]/[account]/index.tsx

```diff
@@ -77,14 +78,20 @@ function ConfigureAccount({ fingerprint, accountIndex, account }: ConfigureAccou
         })}
       >
         <Box pb="3">
-          <AccountCard
-            address={<AccountAddress fingerprint={fingerprint} accountIndex={accountIndex} />}
-            balance={<AccountBalance fingerprint={fingerprint} accountIndex={accountIndex} />}
-            icon={<AvatarIcon color={theme.colors['ink.background-primary']} icon={account.icon} />}
-            name={account.name}
-            key={account.id}
-            iconTestID={defaultIconTestId(account.icon)}
-          />
+          <WalletLoader fingerprint={account.fingerprint} key={account.id}>
```

Comment:
I used `WalletLoader` to get this working but am thinking we could add the name to `account`? Rather than repeat the use of `WalletLoader` ? 

That could cause complications with name updates perhaps